lib/beaker/host/unix/exec.rb

```diff
@@ -50,7 +50,11 @@ def reboot(wait_time=10, max_connection_tries=9, uptime_retries=18)
       sleep wait_time
 
       current_boot_time_str = exec(Beaker::Command.new('who -b'), {:max_connection_tries => max_connection_tries, :silent => true}).stdout
-      current_boot_time = Time.parse(current_boot_time_str.lines.grep(/boot/).first)
```

Comment:
we already do this for the pre-reboot code here https://github.com/puppetlabs/beaker/pull/1659/files#diff-adecec78dcc971d847136c54308d7766R23

Reply:
👍  It should probably be abstracted into a method but this is fine for now.